openstack/cinder/values.yaml

```diff
@@ -72,6 +69,22 @@ cinderApiPortPublic: 443
 cinderApiPortInternal: 8776
 cinderApiPortAdmin: 8776
 
+defaults:
+  backends:
+    common: {}
+    cinder.volume.drivers.netapp.common.NetAppDriver:
```

Comment:
Should we consolidate the key `backend_defaults` from above into this structure (as a follow-up maybe)?

Reply:
Yes, I think, that would make sense. I can create a PR for that.

Reply:
I had another look, and I am not sure, how to integrate it.
The defaults in this section are the defaults for the settings for an individual backend.
The `backend_defaults` are the `backend_defaults` section, which apply to all backends of a host, but only vmware.
It definitely needs some rework, but I am unclear a bit on how to do that.

Reply:
I thought we could maybe do something like `defaults.backends."cinder.volume.drivers.vmware.vmdk.VMwareVcVmdkDriver".backend_defaults` or something - but not sure either. If we don't have a good way, then we leave it until we figure it out $somewhen.